x-pack/plugins/maps/public/classes/sources/es_search_source/get_docvalue_source_fields.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IndexPattern } from '../../../../../../../src/plugins/data/common/index_patterns/index_patterns';
+import { getField } from '../../../../common/elasticsearch_util';
+import { ScriptField } from './es_search_source';
```

Comment:
Should you move ScriptField into this file as well so you don't have a circular dependency where es_search_source imports get_docvalue_source_fields and get_docvalue_source_fields imports es_search_source?